requirements/common.txt

```diff
@@ -47,3 +47,4 @@ opentelemetry-sdk>=1.26.0  # vllm.tracing
 opentelemetry-api>=1.26.0  # vllm.tracing
 opentelemetry-exporter-otlp>=1.26.0  # vllm.tracing
 opentelemetry-semantic-conventions-ai>=0.4.1  # vllm.tracing
+torchao == 0.11.0
```

Comment:
shall we use >= 0.11.0?

Reply:
maybe just in tets: https://github.com/vllm-project/vllm/blob/a09c7ca9f2c01325eb46e82272c55358e5f98e8e/requirements/test.in#L42, https://github.com/vllm-project/vllm/blob/a09c7ca9f2c01325eb46e82272c55358e5f98e8e/requirements/test.txt#L44